scripts/imu_to_pose_node.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+import rclpy
+import rclpy.logging
+from geometry_msgs.msg import PoseStamped
+from rclpy.node import Node
+from rclpy.qos import HistoryPolicy, ReliabilityPolicy, QoSProfile
+from sensor_msgs.msg import Imu
+
+
+class ImuToPoseNode(Node):
+    """
+    Subscribe to the /mavros/imu/data message, get the orientation, and republish it as a
+    PoseStamped message. This is useful for comparing the Pixhawk and x150 outputs in rviz2.
+    """
+
+    def __init__(self):
+        super().__init__('imu_to_pose')
+
+        imu_qos = QoSProfile(reliability=ReliabilityPolicy.BEST_EFFORT,
+                             history=HistoryPolicy.SYSTEM_DEFAULT)
+
+        self._imu_sub = self.create_subscription(Imu, '/mavros/imu/data',
+                                                 self.imu_callback, imu_qos)
+
+        self._pose_pub = self.create_publisher(PoseStamped, 'imu_pose', 10)
+
+    def imu_callback(self, imu_msg: Imu):
+        msg = PoseStamped()
+        msg.header = imu_msg.header
+        msg.pose.orientation = imu_msg.orientation
+
+        # Provide a little offset so that the 2 messages don't stack on top of each other in rviz2.
+        msg.pose.position.x = 0.1
+        msg.pose.position.z = 0.25
```

Comment:
For future-proofing, it would be better here to add 0.1 and 0.25 to the current value of msg.pose.position, in case we eventually have a position.